src/v/cluster/types.h

```diff
@@ -2364,7 +2364,11 @@ struct bootstrap_cluster_cmd_data
 
     auto serde_fields() {
         return std::tie(
-          uuid, bootstrap_user_cred, node_ids_by_uuid, founding_version);
+          uuid,
+          bootstrap_user_cred,
+          node_ids_by_uuid,
+          founding_version,
+          initial_nodes);
```

Comment:
was this already shipped? if it was then we should we bump the version to 3 along adding initial_nodes to serde_fields

Reply:
ok, I've noticed "none - issue does not exist in previous branches"